<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEmailView</class>
 <widget class="QWidget" name="CEmailView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLineEdit" name="emailAddresses">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTreeView" name="groupedEmails">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="byEmailAddress">
     <property name="text">
      <string>Email Addresse(s):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="byFromNames">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="fromNames"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
